portal/templates/redesign/help-and-support_new.html

```diff
@@ -34,29 +37,224 @@ <h1 class="banner__text--primary">Help and support</h1>
     <p class="body-text">Should you experience any difficulties with support for browsers or other devices you are using which you are unable to resolve, please
         <a href="#contact">contact us</a> for assistance.</p>
 </div>
+
 <div id="faqs">
     <div class="background col-sm-8 col-center">
-        <h3>FAQs</h3>
-        <p class="body-text"><strong>+ I am a school student and have forgotten my password. How do I reset it?</strong></p>
-        <p class="body-text"><strong>+ I am a teacher and have forgotten my password. How do I reset it?</strong></p>
-        <p class="body-text"><strong>+ I am an independent student and have forgotten my password. How do I reset it?</strong></p>
-        <p class="body-text"><strong>+ I'm having problems choosing an acceptable password for my account. What am I missing?</strong></p>
-        <p class="body-text"><strong>+ How do I choose a secure and memorable password for my account?</strong></p>
-        <p class="body-text"><strong>+ What is two factor authentication? Do I need it?</strong></p>
-        <p class="body-text"><strong>+ What if I teach at more than one school? Can I sign up with one email address?</strong></p>
-        <p class="body-text"><strong>+ I want to sign up as a Teacher. Why am I asked for my 'professional' email address when registering?</strong></p>
-        <p class="body-text"><strong>+ What should I do when my students leave my school or club?</strong></p>
-        <p class="body-text"><strong>+ What is an independent student?</strong></p>
-        <p class="body-text"><strong>+ What if a student moves to another school? Can they keep their progress?</strong></p>
-        <p class="body-text"><strong>+ What do I need to do to allow an independent student to join my class or school?</strong></p>
-        <p class="body-text"><strong>+ I've set up my students. How do they log in?</strong></p>
-        <p class="body-text"><strong>+ As a student in a school or club, who can I share my custom created levels with?</strong></p>
-        <p class="body-text"><strong>+ As a teacher, who can I share my custom created levels with?</strong></p>
-        <p class="body-text"><strong>+ How can I control the levels my students create?</strong></p>
-        <p class="body-text"><strong>+ I am an independent student. Why can't I share my custom created levels?</strong></p>
+        <h1>FAQs</h1>
+
+<style>
```

Comment:
We'll want to move this `<style>` block to the styles.scss file which deals with all the CSS/styling for the project.
If you're worried about this affecting other inputs/labels throughout the project, you can add a class name specifying that these rules should apply to the inputs and labels specifically in the `faqs` div.